components/Form/index.js

```diff
@@ -122,14 +122,30 @@ function Form({
           required
         />
         <label htmlFor="size">Größe*</label>
-        <StyledInput
-          type="number"
+        <StyledSelect
           name="size"
-          defaultValue={sizeContent}
+          defaultValue={sizeContent ? sizeContent : ""}
```

Comment:
It works correct? Is sizeContent a boolean or is it undefined if it is not set? If it works it's fine for me

Reply:
yes it works. I need it for the dropdown selection so that the value "bitte wählen" is issued